examples/hal/compute/main.rs

```diff
@@ -34,48 +34,54 @@ fn main() {
     #[cfg(any(feature = "vulkan", feature = "dx12", feature = "metal"))]
     let instance = back::Instance::create("gfx-rs compute", 1);
 
-    let adapter = instance.enumerate_adapters().into_iter()
-        .find(|a| a.queue_families
-            .iter()
-            .any(|family| family.supports_compute())
-        )
+    let adapter = instance
+        .enumerate_adapters()
+        .into_iter()
+        .find(|a| {
+            a.queue_families
+                .iter()
+                .any(|family| family.supports_compute())
+        })
         .expect("Failed to find a GPU with compute support!");
 
     let memory_properties = adapter.physical_device.memory_properties();
-    let (mut device, mut queue_group) = adapter
-        .open_with::<_, Compute>(1, |_family| true)
-        .unwrap();
+    let (mut device, mut queue_group) = adapter.open_with::<_, Compute>(1, |_family| true).unwrap();
 
-    let shader = device.create_shader_module(include_bytes!("shader/collatz.spv")).unwrap();
+    let shader = device
+        .create_shader_module(include_bytes!("shader/collatz.spv"))
+        .unwrap();
 
     let (pipeline_layout, pipeline, set_layout, mut desc_pool) = {
         let set_layout = device.create_descriptor_set_layout(
-            &[
-                pso::DescriptorSetLayoutBinding {
-                    binding: 0,
-                    ty: pso::DescriptorType::StorageBuffer,
-                    count: 1,
-                    stage_flags: pso::ShaderStageFlags::COMPUTE,
-                    immutable_samplers: false,
-                }
-            ],
+            &[pso::DescriptorSetLayoutBinding {
+                binding: 0,
+                ty: pso::DescriptorType::StorageBuffer,
+                count: 1,
+                stage_flags: pso::ShaderStageFlags::COMPUTE,
+                immutable_samplers: false,
+            }],
             &[],
         );
 
         let pipeline_layout = device.create_pipeline_layout(Some(&set_layout), &[]);
-        let entry_point = pso::EntryPoint { entry: "main", module: &shader, specialization: &[] };
+        let entry_point = pso::EntryPoint {
+            entry: "main",
+            module: &shader,
+            specialization: &[],
+        };
         let pipeline = device
-            .create_compute_pipeline(&pso::ComputePipelineDesc::new(entry_point, &pipeline_layout))
+            .create_compute_pipeline(&pso::ComputePipelineDesc::new(
+                entry_point,
+                &pipeline_layout,
+            ))
             .expect("Error creating compute pipeline!");
 
         let desc_pool = device.create_descriptor_pool(
             1,
-            &[
-                pso::DescriptorRangeDesc {
-                    ty: pso::DescriptorType::StorageBuffer,
-                    count: 1,
-                },
-            ],
+            &[pso::DescriptorRangeDesc {
```

Comment:
I wonder how this will look like with multiple items in the array

Reply:
It indents it and puts it on a newline, e.g.:
```rust
&[
    struct {
        ..
    },
    struct {
        ..
    },
],
```